amundsen_rds/models/table.py

```diff
@@ -0,0 +1,173 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from sqlalchemy import (
+    BigInteger, Boolean, Column, ForeignKey, Integer, String, Text
+)
+from sqlalchemy.orm import relationship
+
+from amundsen_rds.models.base import (
+    KEY_LEN, NAME_LEN, PUBLISHED_TAG_LEN, URL_LEN, Base
+)
+
+
+class Table(Base):
+    """
+    Table model.
+    """
+    __tablename__ = 'table_metadata'
+
+    rk = Column(String(KEY_LEN), primary_key=True)
```

Comment:
is the rk having the same format as the graph key model (e.g db://cluster.schema/table)?

Reply:
yes, it is same as the node key. It directly use the current node key as record key(rk)